Update install instructions for LGPU (#1037)
PennyLaneAI/pennylane-lightning: README.rst

    The below contains instructions for installing Lightning-GPU ***from source***. For most cases, *this is not required* and one can simply use the installation instructions at `pennylane.ai/install <https://pennylane.ai/install/#high-performance-computing-and-gpus>`__. If those instructions do not work for you, or you have a more complex build environment that requires building from source, then consider reading on.

> Since you will be installing PennyLane-Lightning from the master branch, you will also need to install PennyLane from master:

Reviewer: This is recommended, but it's no longer a must as PR https://github.com/PennyLaneAI/pennylane-lightning/pull/1033 updates the minimum required version of PL to v0.40.0 in the master branch.

> Since you will be installing PennyLane-Lightning from the master branch, you will also need to install PennyLane from master:

Author: ```suggestion
Since you will be installing PennyLane-Lightning from the master branch, it is recommended to install PennyLane from master:
```

    The below contains instructions for installing Lightning-GPU ***from source***. For most cases, *this is not required* and one can simply use the installation instructions at `pennylane.ai/install <https://pennylane.ai/install/#high-performance-computing-and-gpus>`__. If those instructions do not work for you, or you have a more complex build environment that requires building from source, then consider reading on.

Since you will be installing PennyLane-Lightning from the master branch, you will also need to install PennyLane from master:

.. code-block:: bash

    git clone https://github.com/PennyLaneAI/pennylane.git
    cd pennylane

>     pip install .

Reviewer 2: Hey @CatalinaAlbornoz, if you don't need or want to have access to PennyLane files, you can simply go with: `pip install git+https://github.com/PennyLaneAI/pennylane.git@master`

> To install Lightning-GPU from the package sources using the direct SDK path, Lightning-Qubit should be installed before Lightning-GPU (compilation is not necessary):

Reviewer 2: ```suggestion
To install Lightning-GPU from the package sources using the direct SDK path, Lightning-Qubit should be installed first (compilation is not necessary):
```

Reviewer 2: ```suggestion
To install Lightning-GPU from the package sources using the direct SDK path first install Lightning-Qubit (compilation is not necessary):
```
The sentence as is sounds weird at least in my ear. Probably because of the repetition of LGPU. What do you think now? Feel free to suggest other variants.
